workers/data_refinery_workers/processors/utils.py

```diff
@@ -96,6 +102,22 @@ def end_job(job_context: Dict, abort=False):
                 sample.is_processed = True
                 sample.save()
 
+    # S3-sync Original Files
+    for original_files in job_context['original_files']:
+        # Ensure even distribution across S3 servers
+        nonce = ''.join(random.choice(string.ascii_lowercase + string.digits) for _ in range(8)) 
+        result = original_files.sync_to_s3(S3_BUCKET_NAME, nonce + "_" + original_files.filename)
+        if result:
+            original_files.delete_local_file()
+
+    # S3-sync Computed Files
+    for computed_file in job_context['computed_files']:
+        # Ensure even distribution across S3 servers
+        nonce = ''.join(random.choice(string.ascii_lowercase + string.digits) for _ in range(8)) 
+        result = computed_file.sync_to_s3(S3_BUCKET_NAME, nonce + "_" + computed_file.filename)
+        if result:
+            computed_file.delete_local_file()
```

Comment:
I remember you telling me about this trick, but it might be good to include a link to something explaining what and why you're doing it.

Reply:
There is already in a comment in the line above this.